pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -86,6 +87,14 @@ const (
 	// ServiceBrokerRelistBehaviorManual indicates that the broker is only
 	// relisted when the spec of the broker changes.
 	ServiceBrokerRelistBehaviorManual ServiceBrokerRelistBehavior = "Manual"
+
+	// ServiceBrokerRelistRequestsMax is the maximum allowable value for
+	// RelistRequests before it is wrapped around.
+	ServiceBrokerRelistRequestsMax int64 = util.MaxInt64
+
+	// ServiceBrokerRelistRequestsMin is the minimum value that RelistRequests
+	// will be wrapped around to after reaching its maximum
+	ServiceBrokerRelistRequestsMin int64 = util.MinUint64
```

Comment:
You need to add a note to the RelistRequests field explaining the wrapping behavior.